### PR TITLE
DEVX-1192 Adds CONFLUENT prefix values to config.env

### PR DESCRIPTION
**Pull request by rspurgeon**

### CODE REVIEW

**Review by ybyzek — COMMENTED** [review]
@rspurgeon Is this a trial PR or final change?  Should the last 4 lines of this file change?

**Review by andrewegel — APPROVED** [review]
I'll defer to your team's judgement about removing the last 4 lines. Otherwise LGTM

**Comment by rspurgeon** [comment]
> @rspurgeon Is this a trial PR or final change? Should the last 4 lines of this file change?

My plan is to do this incrementally by having the new values put in by automation.  Nothing references them, and I'm not changing anything that references the existing names in this file, so there is no behavioral impact.   After this automation is proven out, we can further modify this file, scripts, etc... to take advantage of these new values.

**Comment by ybyzek** [comment]
> My plan is to do this incrementally by having the new values put in by automation. Nothing references them, and I'm not changing anything that references the existing names in this file, so there is no behavioral impact. After this automation is proven out, we can further modify this file, scripts, etc... to take advantage of these new values.

Makes sense, thanks!

File: utils/config.env
Comment (ybyzek) [inline utils/config.env]: 1. Why is PREVIOUS necessary?
2. Is this by branch or date? e.g. Let's say 5.3.N is released after 5.4.N-1, then what would PREVIOUS be?

File: utils/config.env
Comment (ybyzek) [inline utils/config.env:16]: Are KAFKA and CE_KAFKA switched?

File: utils/config.env
Comment (rspurgeon) [reply utils/config.env]: I'm not certain all of them are necessary, but what was discussed in the companion document was that we would accept all available values from the tools automation here and use what's required as we go.

File: utils/config.env
Comment (rspurgeon) [reply utils/config.env:16]: I'm not certain... @andrewegel does it matter?  I pulled these from the companion spreadsheet.

File: utils/config.env
Comment (rspurgeon) [reply utils/config.env]: @andrewegel can you answer @ybyzek 's question about PREVIOUS?

Comment (ybyzek) [reply utils/config.env]: As we are pulling these values into a public-facing repo, we should have clarity on what they represent.  @andrewegel could you elaborate on the question regarding PREVIOUS?  Thanks!

File: utils/config.env
Comment (andrewegel) [reply utils/config.env:16]: `KAFKA` is correct - `CE_KAFKA` would be `5.4.0-ce-SNAPSHOT` in this context.

File: utils/config.env
Comment (andrewegel) [reply utils/config.env]: `CONFLUENT_PREVIOUS` is just the previous version - take it or leave it, its your choice.

Comment (andrewegel) [reply utils/config.env]: As in this is the table:

`5.4.1` -> `5.4.0`
`5.4.0` -> `` (empty string)

File: utils/config.env
Comment (rspurgeon) [reply utils/config.env:16]: I just hand typed this in so it will be replaced by an appropriate value after we merge this and automation kicks in, is that correct?  Does it need to be a valid value before that happens?

Comment (andrewegel) [reply utils/config.env:16]: The value doesn't really matter, this is an idempotent process, if the script encounters any value, it'll just put in the right one.

File: utils/config.env
Comment (rspurgeon) [reply utils/config.env]: Do you know what specifically defines the value?

Comment (andrewegel) [reply utils/config.env]: https://github.com/confluentinc/release-tools/blob/master/confluent/release/versions.py#L692

Comment (ybyzek) [reply utils/config.env]: @andrewegel thanks for the link.  Based on

```
    Return the previous patch version for the given version without any suffix, or None if this is the first
```

IIUC since there is no 5.5.0 yet, then this should be set to `None`

Comment (rspurgeon) [reply utils/config.env]: I've updated the value here to empty string per @andrewegel comment above

**Review by ybyzek — APPROVED** [review]
> The value doesn't really matter, this is an idempotent process, if the script encounters any value, it'll just put in the right one.

Given this, PR approved.

**Pull request merged**
